.taskcluster.yml

```diff
@@ -153,7 +153,8 @@ tasks:
           export PKG_CONFIG_PATH="/usr/local/opt/zlib/lib/pkgconfig:$PKG_CONFIG_PATH" &&
           export RUSTC_WRAPPER=sccache &&
           (cd wrench && python script/headless.py reftest) &&
-          (cd wrench && cargo build --release)
+          (cd wrench && cargo build --release) &&
+          (cd wrench && cargo run --release -- reftest reftests/clip/fixed-position-clipping.yaml)
```

Comment:
we should just say `cargo run --release -- --precache show reftests/blend/blank.yaml` instead

Reply:
show doesn't terminate, that's why I chose reftest.